### ISSUE TITLE
Multiple copies of @salt-ds installed when using `@salt-ds/data-grid@1.0.1`

### ISSUE THREAD

**Issue by origami-z** [jpmorganchase/salt-ds#1325]
### Package name(s)

Core v1.0.0 (@salt-ds/core), Lab v1.0.0-alpha.0 (@salt-ds/lab), Data Grid v1.0.0 (@salt-ds/data-grid)

### Description

`@salt-ds/data-grid@1.0.1` uses core v1.1.0 and lab v1.0.0-alpha.1 without `^`, which means when used together with newer version of core/lab packages, multiple versions of core/lab are installed.

Is this intentional? 

### Steps to reproduce

Sample `package.json` dependencies 

```
    "@salt-ds/core": "^1.2.0",
    "@salt-ds/data-grid": "^1.0.1",
    "@salt-ds/lab": "^1.0.0-alpha.2",
```

2 copies of core and lab packages are installed, due to data-grid package depend on **exact** version of packages

`yarn info @salt-ds/data-grid@1.0.1` gives

```
  dependencies: {
    clsx: '^1.2.1',
    '@salt-ds/core': '1.1.0',
    '@salt-ds/lab': '1.0.0-alpha.1'
  },
```

### Expected behavior

One copy of core package is needed when using latest version of core, lab, data-grid

### Operating system

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Browser

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

### Are you a JPMorgan Chase & Co. employee?

- [X] I am an employee of JPMorgan Chase & Co.

**Comment by ZarkoPernar** [comment]
How does this happen exactly? I assumed our packages are always published with the same version and dependencies and that this part is automated.

**Comment by origami-z** [comment]
> How does this happen exactly?

See updated issue description

> are always published with the same version and dependencies and that this part is automated.

Yes, so one option is to update the automation to support latest version usage.

---

Also, "Package name(s)" section came from GitHub issue template which needs to be updated to remove version number.

**Comment by origami-z** [comment]
What's the status of this? @ImmanuelBaskaran

**Comment by origami-z** [comment]
Confirmed this is fixed by #1647
